jekyll/_cci2/deployment-integrations.adoc

```diff
@@ -120,11 +120,11 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.4-node # primary container - where job steps are run
+      - image: cimg/ruby:2.4-node # primary container - where job steps are run
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
-      - image: circleci/postgres:9.4.12-alpine # services container
+      - image: cimg/postgres:9.4.12-alpine # services container
```

Comment:
-alpine needs to go.

Reply:
@felicianotech I went ahead and replaced 9.4.12-alpine with 9.6. Is this what you were referring to as the change that was needed?